lang/hi.js

```diff
@@ -1,23 +1,23 @@
 /* eslint quotes: 0 */
 
 export default {
-	"appliedFilters": "Applied Filters:",
-	"noActiveFilters": "No active filters",
-	"filterRemoved": "Filter {filterText} removed",
-	"clearFilters": "Clear filters",
-	"allFiltersRemoved": "All filters removed",
+	"appliedFilters": "लगाए गए फ़िल्टर:",
+	"noActiveFilters": "कोई सक्रिय फ़िल्टर नहीं हैं",
+	"filterRemoved": "फ़िल्टर {filterText} निकाला गया",
+	"clearFilters": "फ़िल्टर साफ़ करें",
+	"allFiltersRemoved": "सभी फ़िल्टर निकाले गए",
 	"categoryTitleMultiple": "{title} ({numSelected})",
-	"searchBy": "Search by {category}",
-	"clear": "Clear",
-	"filter": "Filter",
-	"filterBy": "Filter By",
-	"filterMultiple": "Filter: {numOptions} Filters",
-	"filterSingle": "Filter: 1 Filter",
-	"more": "More",
-	"searchPagedResults": "{rangeStart} - {rangeEnd} of {totalCount} results",
-	"searchPagedResultsForQuery": "{rangeStart} - {rangeEnd} of {totalCount} results for \"{query}\"",
-	"searchResults": "{totalCount} results",
-	"searchResultsForQuery": "{totalCount} results for \"{query}\"",
-	"sort": "Sort",
-	"sortWithOption": "Sort: {option}"
+	"searchBy": "{category} के अनुसार खोजें",
+	"clear": "साफ़ करें",
+	"filter": "फ़िल्टर करें",
+	"filterBy": "इसके अनुसार फ़िल्टर करें",
+	"filterMultiple": "फ़िल्टर करें: {numOptions} फ़िल्टर",
+	"filterSingle": "फ़िल्टर: 1 फ़िल्टर",
+	"more": "अधिक",
+	"searchPagedResults": "{rangeStart} - {totalCount} में से {rangeEnd} परिणाम",
+	"searchPagedResultsForQuery": "{rangeStart} - \"{query}” के लिए {totalCount} में से {rangeEnd} परिणाम",
+	"searchResults": "{totalCount} परिणाम",
+	"searchResultsForQuery": "\"{query}” के लिए {totalCount} परिणाम",
```

Comment:
```suggestion
	"searchResultsForQuery": "“{query}” के लिए {totalCount} परिणाम",
```
Seems like we should maybe be consistent here for the quotation marks?  (Either switch to this or switch it back to the other style?)